service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1537,6 +1541,53 @@ components:
           items:
             $ref: "#/components/schemas/Filter"
 
+    OccurrenceForPrimaryFilter:
+      type: object
+      required: [entityGroup, occurrenceEntity]
+      description: |
+        Filters an occurrence entity with a criteria-occurrence entity group (e.g. condition
+        occurrences for people with gender=female, condition=diabetes).
+      properties:
+        entityGroup:
+          type: string
+        occurrenceEntity:
+          type: string
+        primarySubfilter:
+          $ref: "#/components/schemas/Filter"
+
+    PrimaryWithCriteriaFilter:
+      type: object
+      required: [entityGroup]
+      description: |
+        Filter the primary entity with a criteria-occurrence entity group (e.g. people where 
+        condition=diabetes and visit_type=inpatient).
+        If the group_by_count_operator is set, then the group_by_count_value must also be set.
```

Comment:
You should probably use the camelCase names (e.g. `groupByCountOperator`) to match the names in the file.